docs/OV_Runtime_UG/supported_plugins/GPU_Extensibility.md

```diff
@@ -0,0 +1,233 @@
+# How to Implement Custom GPU Operations {#openvino_docs_OV_UG_supported_plugins_GPU_Extensibility}
```

Comment:
do we need to provide a link from the common OV extensibility? Or otherwise - return the document to `extennding OpenVINO` and from GPU plugin's page just refer to it. Otherwise, we are mixing user documentation with "basic" stuff and advanced guides with extensions.
CC @ilyachur